web/src/message_store.ts

```diff
@@ -55,6 +55,7 @@ export type RawMessage = {
     sender_email: string;
     sender_full_name: string;
     sender_id: number;
+    condensed?: boolean;
```

Comment:
I think this is the wrong type to add this to; does the server ever return this field? I think you want the subtype that's actually used in your module.